idaes/power_generation/unit_models/helm/splitter.py

```diff
@@ -158,12 +158,12 @@ def sum_split(b, t):
         @self.Constraint(time, self.outlet_list, doc="Pressure constraint")
         def pressure_eqn(b, t, o):
             o_block = getattr(self, "{}_state".format(o))
-            return self.mixed_state[t].pressure == o_block[t].pressure
+            return self.mixed_state[t].pressure*1e-6 == o_block[t].pressure*1e-6
```

Comment:
Not sure if this is standard for the Helmholtz models, but should we be hard coding these scaling factors, or using the scaling tools?

Reply:
Actually now that I looked at the rest of the model, this is in the ```calculate_scale_factors``` method.  That means if someone uses that the equations will be double scaled, so we should probably not make these splitter scaling changes.  Let me know if the splitter scaling isn't working for whatever reason and I'll take a harder look at it.